dftimewolf/cli/dftimewolf_recipes.py

```diff
@@ -134,11 +141,11 @@ def _LoadConfigurationFromFile(self, configuration_file_path):
     """
     try:
       if config.Config.LoadExtra(configuration_file_path):
-        sys.stderr.write('Configuration loaded from: {0:s}\n'.format(
+        logger.debug('Configuration loaded from: {0:s}'.format(
             configuration_file_path))
 
     except errors.BadConfigurationError as exception:
-      sys.stderr.write('{0!s}'.format(exception))
+      logger.warning('{0!s}'.format(exception))
```

Comment:
is this a `warning` or an `error` ?

also if you want to include the stack trace you can also just simply do:

```
except errors.BadConfigurationError:
  logger.warning('Bad configuration value, with error:', exc_info=True)
```

Reply:
In this case it's a warning, since we'd like to continue with defaults. Good tip about the stack trace! Do you know if it's part of the same message or is it split by lines?

Reply:
it's split by lines (unfortunately)

Reply:
That's good to know, thanks. In my case I don't want to log the stack trace at the same time as the error, so I store it somewhere for later use.